### PR TITLE
refactor: remove unused AAD code

### PR DESCRIPTION
**Pull request by blackchoey**

### CODE REVIEW

File: packages/fx-core/src/component/debugHandler/sso.ts [hunk 1]
```diff
@@ -252,11 +249,9 @@
       });
 
       const context = this.constructPluginContext(this.envInfoV3, this.cryptoProvider);
-      const manifest = await AadAppManifestManager.loadAadManifest(context);
+      const manifest = {};
```
[the remark this answers is not in the document]
Comment (blackchoey) [reply packages/fx-core/src/component/debugHandler/sso.ts:252]: The entire file will be removed later, ignore this alert

File: packages/fx-core/src/component/debugHandler/sso.ts [hunk 2]
```diff
@@ -311,8 +306,7 @@
       ].oauthAuthority = `${Constants.oauthAuthorityPrefix}/${TokenProvider.tenantId}`;
 
       const context = this.constructPluginContext(this.envInfoV3!, this.cryptoProvider!);
-      const manifest = await AadAppManifestManager.loadAadManifest(context);
-      await AadAppClient.updateAadAppUsingManifest(TelemetryEvent.DebugSetUpSSO, manifest, false);
+      const manifest = {};
```
[the remark this answers is not in the document]
Comment (blackchoey) [reply packages/fx-core/src/component/debugHandler/sso.ts:309]: The entire file will be removed later, ignore this alert

File: packages/fx-core/src/component/debugHandler/sso.ts [hunk 3]
```diff
@@ -323,8 +317,7 @@
   private async buildAndSaveAADManifest(): Promise<Result<string[], FxError>> {
     try {
       const context = this.constructPluginContext(this.envInfoV3!, this.cryptoProvider!);
-      const manifest = await AadAppManifestManager.loadAadManifest(context);
-      await AadAppManifestManager.writeManifestFileToBuildFolder(manifest, context);
+      const manifest = {};
```
[the remark this answers is not in the document]
Comment (blackchoey) [reply packages/fx-core/src/component/debugHandler/sso.ts:320]: The entire file will be removed later, ignore this alert

**Review by kuojianlu — APPROVED** [review]

**Review by swatDong — APPROVED** [review]
Lucky PR number~ :D

**Review by sffamily — APPROVED** [review]
Good PR number!

**Review by SLdragon — APPROVED** [review]

**Review by KennethBWSong — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Comment by blackchoey** [comment]
E2E test: https://github.com/OfficeDev/TeamsFx/actions/runs/5151154188
The result is same with today's scheduled E2E test run

**Pull request merged**